README.md

```diff
@@ -55,7 +55,7 @@ Currently, textSQL only supports visualizing zipcodes and cities on the map usin
 
 #### Datasets: 
 
-A lot of the users of this project have asked for historical census data (trends), weather, health, transportation and real-estate data. Feel free to create a pull request or drop a link to your dataset in this [Discord](https://discord.gg/JZtxhZQQus). 
+A lot of the users of this project have asked for historical census data (trends), weather, health, transportation and real-estate data. Feel free to create a pull request or contribute a dataset via our [dedicated submission form](https://airtable.com/shrDKRRGyRCihWEZd).
```

Comment:
can we keep the discord link too. no one has submitted anything via Airtable yet

Reply:
Was debating the same in my head-- thank you. Readded!